.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This file can be removed i believe

Reply:
mistaken artifact from scaffolding the project. It should have been .eslintrc and the contents too were different. I have since fix this issue.

Reply:
but this file is at the root, shouldn't it be removed?